docker/.env

```diff
@@ -0,0 +1,17 @@
+#HUGGING_FACE_HUB_TOKEN=<secret>
```

Comment:
The example dotenv file already exists, is this necessary?

Reply:
I've removed the additional example (example.env), that wasn't really necessary.

The real .env (with commented-out properties) remains, as it's necessary for docker compose to work (without it, it complains ".env: no such file or directory").